src/core/ext/client_channel/client_channel_factory.h

```diff
@@ -35,13 +35,19 @@
 #define GRPC_CORE_EXT_CLIENT_CHANNEL_CLIENT_CHANNEL_FACTORY_H
 
 #include <grpc/impl/codegen/grpc_types.h>
+#include <grpc/support/avl.h>
 
 #include "src/core/ext/client_channel/subchannel.h"
 #include "src/core/lib/channel/channel_stack.h"
+#include "src/core/lib/transport/mdstr_hash_table.h"
 
 // Channel arg key for client channel factory.
 #define GRPC_ARG_CLIENT_CHANNEL_FACTORY "grpc.client_channel_factory"
 
+// Channel arg key for the mapping of LB server addresses to their names for
+// secure naming purposes.
+#define GRPC_ARG_GRPCLB_BALANCER_NAMES "grpc.grpclb_server_to_balancer_names"
```

Comment:
Moved to client_channel_factory.c. Everything relies on `grpc_channel_targets_info_create_channel_arg` and `grpc_channel_targets_info_find_in_args` now.